manifests/tftp/netboot/params.pp

```diff
@@ -8,10 +8,15 @@
       if versioncmp($::operatingsystemmajrelease, '6') <= 0 {
         $grub_installation_type = 'redhat_old'
         $packages = ['grub']
-      } elsif versioncmp($::operatingsystemrelease, '7.4') >= 0 and $::operatingsystem != 'Fedora' {
-        # RHEL 7.4 renamed the packages and introduced a regression in the MAC loading of configs
+      } elsif versioncmp($::operatingsystemrelease, '7.4') == 0 and $::operatingsystem != 'Fedora' {
```

Comment:
I think CentOS has some versions that append a `.z` number where this would break. OTOH, CentOS doesn't support individual .y releases so I'd be fine with breaking that.

Reply:
Yes, I'd rather not handle that case, this would only affect very older CentOS 7's.